### PR TITLE
disable test case

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinPortalNavigationEditNavigationTestIT.java [hunk 1]
```diff
@@ -51,6 +53,7 @@ public void setupBeforeMethod() {
    * <li>Test Case Name:Edit node.</li>
    */
   @Test
+  @Disabled
```
Comment (mgreau) [inline gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinPortalNavigationEditNavigationTestIT.java]: This is not due to a bug in PLF?

Comment (ilyeskallala) [inline gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinPortalNavigationEditNavigationTestIT.java]: yes it is due to a bug in PLF but we didn't create an issue in JIRA until we find the cause of the bug

Comment (mgreau) [inline gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinPortalNavigationEditNavigationTestIT.java]: OK so please add the annotation with this information `@BugInPLF("No issue")`

Comment (ilyeskallala) [inline gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinPortalNavigationEditNavigationTestIT.java]: done

**Pull request merged**